backend/sirius-web-spring-collaborative-selection/src/main/java/org/eclipse/sirius/web/spring/collaborative/selection/SelectionEventProcessor.java

```diff
@@ -0,0 +1,200 @@
+
+    private final IEditingContext editingContext;
+
+    private final SelectionDescription selectionDescription;
+
+    private final UUID selectionId;
+
+    private final Object object;
+
+    private final List<ISelectionEventHandler> selectionEventHandlers;
+
+    private final ISubscriptionManager subscriptionManager;
+
+    private final Many<IPayload> sink = Sinks.many().multicast().directBestEffort();
+
+    private final Many<Boolean> canBeDisposedSink = Sinks.many().unicast().onBackpressureBuffer();
+
+    private final AtomicReference<Selection> currentSelection = new AtomicReference<>();
+
+    public SelectionEventProcessor(IEditingContext editingContext, SelectionDescription selectionDescription, UUID selectionId, Object object, List<ISelectionEventHandler> selectionEventHandlers,
+            ISubscriptionManager subscriptionManager) {
+        this.logger.trace("Creating the selection event processor {}", selectionId); //$NON-NLS-1$
+
+        this.selectionDescription = Objects.requireNonNull(selectionDescription);
+        this.editingContext = Objects.requireNonNull(editingContext);
+        this.selectionId = Objects.requireNonNull(selectionId);
+        this.object = Objects.requireNonNull(object);
+        this.selectionEventHandlers = Objects.requireNonNull(selectionEventHandlers);
+        this.subscriptionManager = Objects.requireNonNull(subscriptionManager);
+
+        Selection selection = this.refreshSelection();
+        this.currentSelection.set(selection);
+
+    }
+
+    @Override
+    public IRepresentation getRepresentation() {
+        return this.currentSelection.get();
+    }
+
+    @Override
+    public ISubscriptionManager getSubscriptionManager() {
+        return this.subscriptionManager;
+    }
+
+    @Override
+    public Optional<EventHandlerResponse> handle(IRepresentationInput representationInput) {
+        Optional<EventHandlerResponse> result = Optional.empty();
+        if (representationInput instanceof ISelectionInput) {
+            ISelectionInput selectionInput = (ISelectionInput) representationInput;
+
+            Optional<ISelectionEventHandler> optionalSelectionEventHandler = this.selectionEventHandlers.stream().filter(handler -> handler.canHandle(selectionInput)).findFirst();
+
+            if (optionalSelectionEventHandler.isPresent()) {
+                ISelectionEventHandler selectionEventHandler = optionalSelectionEventHandler.get();
+                EventHandlerResponse eventHandlerResponse = selectionEventHandler.handle(this.currentSelection.get(), selectionInput);
+
+                this.refresh(representationInput, eventHandlerResponse.getChangeDescription());
+
+                result = Optional.of(eventHandlerResponse);
+            } else {
+                this.logger.warn("No handler found for event: {}", selectionInput); //$NON-NLS-1$
+            }
+        }
+
+        return result;
```

Comment:
While it's not a major problem, you could remove all of this code since you do not have any action to handle. In a similar fashion, you can remove the selection event handlers.